core/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java

```diff
@@ -47,12 +41,13 @@ public void testOsStats() {
         OsStats stats = probe.osStats();
         assertNotNull(stats);
         assertThat(stats.getTimestamp(), greaterThan(0L));
+        assertThat(stats.getCpu().getPercent(), anyOf(equalTo((short) -1), is(both(greaterThanOrEqualTo((short) 0)).and(lessThanOrEqualTo((short) 100)))));
```

Comment:
It would be nice to explicitly test on which platform getSystemCpuLoad returns -1, if any, like it is for the load average